src/main/scala/com/yotpo/metorikku/code/steps/ToAvro.scala

```diff
@@ -0,0 +1,43 @@
+package com.yotpo.metorikku.code.steps
+
+import com.yotpo.metorikku.exceptions.MetorikkuException
+import org.apache.spark.sql.functions.struct
+import za.co.absa.abris.avro.functions.to_avro
+import za.co.absa.abris.avro.read.confluent.SchemaManager
+
+object ToAvro {
+
+  val message = "You need to send the following parameters to output to Avro format:" +
+    "table, schema.registry.url, schema.registry.topic, value.schema.name, value.schema.namespace, kafka.url " +
+    "Will create an entry in the schema registry under: <schema.registry.topic>-<value.schema.namespace>.<value.schema_name>"
+
+  def run(ss: org.apache.spark.sql.SparkSession, metricName: String, dataFrameName: String, params: Option[Map[String, String]]): Unit = {
+    params match {
+      case Some(parameters) => {
+        val table = parameters.get("table").get
```

Comment:
Maybe wrap in try-catch and raise a proper exception when a parameter is missing.

Reply:
:\ good idea